fe/fe-core/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -1651,4 +1651,8 @@ public class Config extends ConfigBase {
 
     @ConfField
     public static boolean enable_vectorized_load = false;
+
+    @ConfField(mutable = false, masterOnly = true)
+    public static int backend_rpc_timeout_ms = 60000; // 1 min
```

Comment:
Please add comment for this config and also modify the config document
`admin-manual/config/fe-config.md`